src/core/common/trickle_timer.hpp

```diff
@@ -109,6 +109,36 @@ class TrickleTimer : public TimerMilli
      */
     Mode GetMode(void) const { return mMode; }
 
+    /**
+     * This method gets the current interval min value of the trickle timer.
+     *
+     * @returns The current interval min value of the trickle timer.
+     */
+    uint32_t GetIntervalMin(void) const { return mIntervalMin; }
```

Comment:
nit: is this used somewhere?

Reply:
No, I'll remove. I had been using the max one and had this for symmetry.